src/components/views/elements/TagComposer.tsx

```diff
@@ -64,6 +67,17 @@ export default class TagComposer extends React.PureComponent<IProps, IState> {
         this.props.onRemove(tag);
     }
 
+    private onValidateKeyword = async (fieldState: IFieldState) => {
+        if (!this.props.onValidate) {
+            this.setState({ isNewTagValid: true });
+            return { };
```

Comment:
we don't generally return empty objects - is it required here, or can we use `null`?